src/components/button/button.component.js

```diff
@@ -22,7 +22,6 @@ const Button = React.forwardRef((props, ref) => {
       iconType={ iconType }
       iconPosition={ iconPosition }
       variant={ theme }
-      as={ (props.href || props.to) ? 'a' : 'button' }
```

Comment:
Hey @grabkowski thanks for carrying the work on while I was away, just curious as to why you made these changes? Might be missing something obvious as my mind still thinks it's on holiday 🍸